src/CSSmodule/bookitem.css

```diff
@@ -0,0 +1,144 @@
+.bookList {
+  margin: 2rem 3rem;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.bookSection {
+  display: flex;
+  width: inherit;
+  justify-content: space-between;
+  padding: 10px;
+  background-color: #fff;
+}
+
+.userContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.detail {
+  display: flex;
+  flex-direction: column;
+}
+
+.titles {
+  font-size: 42px;
+}
+
+.category {
+  color: rgb(58, 58, 59);
+}
+
+.author {
+  color: #227ded;
+}
```

Comment:
- [ ] Please kindly ensure that you stick to the design guidelines(font weights, font-size, spacing, font type, etc)

### Your design
![image](https://user-images.githubusercontent.com/43995830/208231742-fcf63f97-5eb1-4422-a530-a0b9895b86b1.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/208231723-19d0816c-aafb-43c6-b7c8-9c1eafb12f22.png)